packages/@ember/-internals/owner/index.ts

```diff
@@ -208,12 +218,18 @@ export interface Factory<T extends object> {
   // able to provide a *narrower* interface than "exactly the public fields on
   // the class" while still falling back to the "exactly the public fields on
   // the class" for the general case. :sigh:
+  //
+  // We stills upply both signatures because even though the second one means
```

Comment:
```suggestion
  // We still supply both signatures because even though the second one means
```

Reply:
EWWWWW, what did I write?!?

Reply:
Ah, but I deleted this separately because I couldn't make it work (as you found on a later commit).